src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/infoboxbuilder/SpecialInfoboxBuilderPageObject.java

```diff
@@ -0,0 +1,239 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.special.infoboxbuilder;
+
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.elements.oasis.pages.TemplatePage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.PortableInfoboxPageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.special.SpecialPageObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+/**
+ * @ownshership: Content West-Wing
+ */
+public class SpecialInfoboxBuilderPageObject extends SpecialPageObject {
+
+  private int DEFAULT_TITLES = 1;
+  private int DEFAULT_IMAGES = 1;
+  private int DEFAULT_ROWS = 2;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this field "DEFAULT_ROWS" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00116)